pkgs/by-name/mo/modrinth-app/package.nix

```diff
@@ -65,6 +67,10 @@ symlinkJoin {
 
   postBuild = ''
     gappsWrapperArgs+=(
+      # Fix GLib-GIO-ERROR **: No GSettings schemas are installed on the system
+      --prefix XDG_DATA_DIRS : "${gsettings-desktop-schemas}/share/gsettings-schemas/${gsettings-desktop-schemas.name}"
+      # Fix GLib-GIO-ERROR **: Settings schema 'org.gtk.Settings.FileChooser' is not installed
+      --prefix XDG_DATA_DIRS : "${gtk3}/share/gsettings-schemas/${gtk3.name}"
```

Comment:
You would usually add the derivations of the missing schemas to `buildInputs`.

I am sketical about `gtk3` being here, as we use wrapGAppsHook4 which should already include `gtk4`.